Cargo.lock

```diff
@@ -88,6 +97,15 @@ version = "0.1.2"
 source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "fbf688625d06217d5b1bb0ea9d9c44a1635fd0ee3534466388d18203174f4d11"
 
+[[package]]
+name = "android_system_properties"
```

Comment:
even more weird deps

Reply:
there actually used to be a bug with logging breaking with 1.28 tracing, did you run this version to and check that all logs look normal?
Including the `[Relaychain]` `[Parachain]` prefixes

Reply:
![logs-0 9 28](https://user-images.githubusercontent.com/70108835/211402124-16ca362f-8f53-44f2-85df-2f1878ab4567.png)

Reply:
Nothing weird from what I can tell